webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2460,6 +2463,43 @@ var CURRENT_GROUP_DATA={};
                 mck_text_box.addEventListener("keyup",function(){
                     _this.toggleMediaOptions(this);
 
+                });
+                $mck_text_box.keyup(function (){
+                    if(CHAR_CHECK){
+                        var warning_box = document.getElementsByClassName('mck-char-warning')[0];
+                        var char_count = document.getElementsByClassName('mck-char-count')[0];
+                        var str = ($mck_text_box.text().toString()).trim();
+                        var len = str.length;
+                        if(len > 199){
+                            if(len > 256){
+                                var rem = len - 256; 
+                                var remtxt ="(Remove " + rem + " characters)";
```

Comment:
@hardikparashar where is 'remtxt' local variable of if block used?

Reply:
also is rem required?

Reply:
rem removed and remtxt declaration changed in new commit